src/core/agent/services/keriaNotificationService.ts

```diff
@@ -835,6 +839,15 @@ class KeriaNotificationService extends AgentService {
   onLongOperationComplete(callback: (event: OperationPendingEvent) => void) {
     this.props.eventEmitter.on(EventTypes.Operation, callback);
   }
+
+  onAddPendingOperation() {
```

Comment:
This is actually something that this service needs for the core to operate, and not something that the UI needs to register in AppWrapper.

Can we remove this function and call the eventEmitter in the constructor?